gapic/src/main/com/google/gapid/perfetto/views/SliceSelectionView.java

```diff
@@ -77,6 +80,29 @@ public SliceSelectionView(Composite parent, State state, SliceTrack.Slice slice)
       createLabel(main, slice.name);
     }
 
+    RenderStageDebugMarkers debugMarkers = slice.getDebugMarkers();
+    if (debugMarkers != null) {
+      LinkedHashMap<String, String> props = new LinkedHashMap<String, String>();
+      if (debugMarkers.frameBufferHandle != 0 || !debugMarkers.frameBufferName.isEmpty()) {
+        props.put("VkFrameBuffer:", debugMarkers.frameBufferName + "<" + String.format("0x%08X", debugMarkers.frameBufferHandle) + ">");
+      }
+      if (debugMarkers.renderPassHandle != 0 || !debugMarkers.renderPassName.isEmpty()) {
+        props.put("VkRenderPass:", debugMarkers.renderPassName + "<" + String.format("0x%08X", debugMarkers.renderPassHandle) + ">");
+      }
+      if (debugMarkers.commandBufferHandle != 0 || !debugMarkers.commandBufferName.isEmpty()) {
+        props.put("VkCommandBuffer:", debugMarkers.commandBufferName + "<" + String.format("0x%08X", debugMarkers.commandBufferHandle) + ">");
+      }
+
+      if (!props.isEmpty()) {
+        withLayoutData(createBoldLabel(main, "Debug Markers"),
```

Comment:
Should be named something like "GPU Info" instead of "Debug Markers"

Reply:
Done